airflow-alt-ldap/auth/backend/ldap_auth.py

```diff
@@ -257,20 +259,38 @@ def login(self, request):
                            form=form)
 
     try:
-        LdapUser.try_login(username, password)
+        ldap_user_entry = LdapUser.try_login(username, password)
         LOG.info("User %s successfully authenticated", username)
 
         session = settings.Session()
+
+        if configuration.has_option("ldap", "airflow_user_attr"):
+            try:
+                airflow_username = ldap_user_entry[configuration.get("ldap", "airflow_user_attr")]
+            except KeyError as e:
+                LOG.error("Could not value for airflow_user_attribute from LDAP. Please check airflow_user_attr "
+                          "in airflow.cfg")
```

Comment:
this multi-line concatenation works in python, but it's better to merge the two lines, as it's less readable and probably incidental.